.swiftlint.yml

```diff
@@ -0,0 +1,18 @@
+# Set max line length before warning (default is 120)
+line_length: 240
+
+disabled_rules:
```

Comment:
Let's rework the `disabled_rules` to work on a folder basis. I don't think we should generally allow exceptions for `force_try`, `force_cast`, or `type_name`. Not sure what `statement_position` means. For reference, here's how we had the old [global .swiftlint.yml](https://github.com/BottleRocketStudios/iOS-Hyperspace/blob/fe948aebe2f1553e84a089183142df9a4c1012b9/Example/.swiftlint.yml) and the [tests .swiftlint.yml](https://github.com/BottleRocketStudios/iOS-Hyperspace/blob/fe948aebe2f1553e84a089183142df9a4c1012b9/Example/Tests/.swiftlint.yml).